simplq/src/components/pages/QueueHistory/AddedInQueue.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import AddIcon from '@material-ui/icons/Add';
+import styles from '../../common/SidePanel/SidePanel.module.scss';
+import itemStyles from './QueueHistory.module.scss';
+
+export default ({ name, tokenNumber, creationTime, tokenCreationTimestamp }) => {
```

Comment:
I feel it would be better have a single HistoryEvent item that can take as parameter if it's a "ADD" or a "REMOVE", what do you think?
```
tokenName, tokenNumber, timestamp, action
```

Reply:
Made this into a single file. Resolved in the latest PR